stdlib/public/runtime/CompatibilityOverride.def

```diff
@@ -138,15 +138,15 @@ OVERRIDE_KEYPATH(getKeyPath, const HeapObject *, , , swift::,
                  (const void *pattern, const void *arguments),
                  (pattern, arguments))
 
-OVERRIDE_METADATALOOKUP(getTypeByMangledNode, TypeInfo, , SWIFT_CC(swift), swift::,
+OVERRIDE_METADATALOOKUP(getTypeByMangledNode, TypeLookupErrorOr<TypeInfo>, , SWIFT_CC(swift), swift::,
```

Comment:
Is this safe?  Changing the return type of an ABI function seems risky.

Reply:
This isn't ABI, it's just a compatibility override point, and compatibility overrides are per-version already. `RuntimeFunctions.def` is where the ABI lives.